### PR TITLE
HacktoberFest: Added a solution for Project Euler problem_024

### PR DESCRIPTION
**Pull request by GantaChaitra**
### **Describe your change:**

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized. 
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by GantaChaitra** [comment]
Hey can you please help me out ! .. I am a beginner and I am not understanding how to fix this ..

**Comment by poyea** [comment]
Thanks for submitting. Please follow the instruction in `project_euler`. There's a readme on the requirement for parameters.

File: project_euler/problem_024/sol2.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+"""A permutation is an ordered arrangement of objects. For example, 3124 is one possible permutation of the digits 1, 2, 3 and 4. If all of the permutations are listed numerically or alphabetically, we call it lexicographic order. The lexicographic permutations of 0, 1 and 2 are:
```
Comment (poyea) [inline project_euler/problem_024/sol2.py:1]: Please split this line:
`project_euler/problem_024/sol2.py:1:89: E501 line too long (280 > 88 characters)`

File: project_euler/problem_024/sol2.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+012   021   102   120   201   210
+What is the millionth lexicographic permutation of the digits 0, 1, 2, 3, 4, 5, 6, 7, 8 and 9?
```
Comment (poyea) [inline project_euler/problem_024/sol2.py:4]: Please split this line too:
`project_euler/problem_024/sol2.py:4:89: E501 line too long (94 > 88 characters)`

Comment (GantaChaitra) [reply project_euler/problem_024/sol2.py]: Thank you so much @poyea !!

File: project_euler/problem_024/sol2.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
+"""A permutation is an ordered arrangement of objects.
+For example, 3124 is one possible permutation of the digits 1, 2, 3 and 4.
+If all of the permutations are listed numerically or alphabetically
+, we call it lexicographic order.
```
Comment (poyea) [inline project_euler/problem_024/sol2.py:4]: better put the comma to the previous line

File: project_euler/problem_024/sol2.py [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    '2783915460'
+ok
+1 items had no tests:
+    sol2
+1 items passed all tests:
+   1 tests in sol2.solution
+1 tests in 2 items.
+1 passed and 0 failed.
+Test passed.
+"""
```
Comment (poyea) [inline project_euler/problem_024/sol2.py:42]: this is not very necessary

File: project_euler/problem_024/sol2.py (hunk 3 above)
Comment (GantaChaitra) [reply project_euler/problem_024/sol2.py:4]: okay !

File: project_euler/problem_024/sol2.py [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+from itertools import permutations
+
+
+def solution():
+    """
+    >>> solution()
+    '2783915460'
+    """
+    lexi_perm = "".join(list(permutations("0123456789"))[999999])
```
Comment (poyea) [inline project_euler/problem_024/sol2.py:21]: We can make `999999` as the default parameter.

File: project_euler/problem_024/sol2.py [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+012   021   102   120   201   210
+
+What is the millionth lexicographic permutation of the digits
+0, 1, 2, 3, 4, 5, 6, 7, 8 and 9?
+
+"""
+from itertools import permutations
+
+
+def solution():
```
Comment (poyea) [inline project_euler/problem_024/sol2.py:16]: Please add some type hints.

File: project_euler/problem_024/sol2.py (hunk 4 above)
Comment (GantaChaitra) [reply project_euler/problem_024/sol2.py:42]: sure , i will remove those lines :)

File: project_euler/problem_024/sol2.py (hunk 5 above)
Comment (GantaChaitra) [reply project_euler/problem_024/sol2.py:21]: i dont want to add one more line of code

**Review by BryanLaura — APPROVED** [review]

**Comment by GantaChaitra** [comment]
Hey ... I will change code and push it again after adding some type hints .

**Review by BryanLaura — APPROVED** [review]

**Review by BryanLaura — APPROVED** [review]

**Review by BryanLaura — APPROVED** [review]

**Review by BryanLaura — APPROVED** [review]

**Review by BryanLaura — APPROVED** [review]

**Review by BryanLaura — APPROVED** [review]

**Review by BryanLaura — APPROVED** [review]

**Comment by dhruvmanila** [comment]
Hey, thank you for your contribution but we're not accepting solutions to existing problems, maybe try solving a problem that doesn't exist in the directory? Thank you for your cooperation.

**Pull request closed without merging**